Feature/prepare for opam repo

mikonieminen opened the pull request.

mikonieminen committed:
Add license information

mikonieminen committed:
Add CircleCI integration

mikonieminen committed:
Update project meta data

joseferben commented on Makefile:
```diff
@@ -0,0 +1,8 @@
+	@dune clean
+
+test-all:
+	SIHL_ENV=test dune runtest --force --no-buffer test
```
@mikonieminen We can remove `SIHL_ENV` imo

mikonieminen replied:
oops

mikonieminen replied:
fixed

jerben commented on README.md:
```diff
@@ -45,3 +45,9 @@ And finally review that the email is correctly generated in the service:
+
+## License
+
+Copyright (c) 2020 Miko Nieminen
```
@mikonieminen Should we put the same claim as in Sihl "Oxidizing Systems"?

joseferben commented on .circleci/config.yml:
```diff
@@ -0,0 +1,40 @@
+  build:
+    docker:
+      - image: ocaml/opam2:4.08
+        environment:
```
@mikonieminen This looks wrong

mikonieminen replied:
It's copied from sihl

joseferben replied:
Hmm I think you are right

mikonieminen committed:
Pin dev version of colombe

To run out tests that use ethereal.email service, we need unreleased
version of colombe. This pins the needed version in the build env.

joseferben commented:
Approved.
